api/handler/handler.go

```diff
@@ -43,6 +43,6 @@ func (h *Handler) DescribeDepartments(w http.ResponseWriter, r *http.Request) {
 }
 
 type department struct {
-	Name     string              `json:"name"`
-	Metadata []map[string]string `json:"metadata"`
+	Name     string              		`json:"name"`
```

Comment:
since `h.db.SeattleOfficerMetadata()` and `h.db.TacomaOfficerMetadata()` both return `data.DepartmentMetadata`, we dont need to declare a `handler` level `department struct`

lines 29-38 can be replaced with 
```
departments := []data.DepartmentMetadata{h.db.SeattleOfficerMetadata(), h.db.TacomaOfficerMetadata()
```